jarbas/core/tests/test_reimbursements_command.py

```diff
@@ -89,69 +89,67 @@ def test_serializer(self):
+
+
+class TestMarkNonUpdated(TestCommand):
+
+    @patch.object(Reimbursement.objects, 'filter')
+    def test_mark_available_in_latest_dataset(self, filter_):
+        self.command.started_at = 42
+        self.command.mark_not_updated_reimbursements()
+        filter_.assert_called_once_with(last_update__lt=self.command.started_at)
+        filter_.return_value.update.assert_called_once_with(available_in_latest_dataset=False)
 
 
 class TestConventionMethods(TestCommand):
 
     @patch('jarbas.core.management.commands.reimbursements.print')
     @patch('jarbas.core.management.commands.reimbursements.Command.reimbursements')
-    @patch('jarbas.core.management.commands.reimbursements.Command.bulk_create_by')
+    @patch('jarbas.core.management.commands.reimbursements.Command.create_or_update')
     @patch('jarbas.core.management.commands.reimbursements.Command.print_count')
-    def test_handler_without_options(self, print_count, bulk_create_by, reimbursements, print_):
+    @patch('jarbas.core.management.commands.reimbursements.Command.mark_not_updated_reimbursements')
+    def test_handler_without_options(self, mark, print_count, create, reimbursements, print_):
         reimbursements.return_value = (1, 2, 3)
-        self.command.handle(dataset='reimbursements.xz', batch_size=42)
+        self.command.handle(dataset='reimbursements.xz')
         print_.assert_called_once_with('Starting with 0 reimbursements')
-        bulk_create_by.assert_called_once_with(reimbursements, 42)
+        create.assert_called_once_with(reimbursements)
         self.assertEqual('reimbursements.xz', self.command.path)
+        mark.assert_called_once_with()
 
     @patch('jarbas.core.management.commands.reimbursements.print')
     @patch('jarbas.core.management.commands.reimbursements.Command.reimbursements')
-    @patch('jarbas.core.management.commands.reimbursements.Command.bulk_create_by')
+    @patch('jarbas.core.management.commands.reimbursements.Command.create_or_update')
     @patch('jarbas.core.management.commands.reimbursements.Command.drop_all')
     @patch('jarbas.core.management.commands.reimbursements.Command.print_count')
-    def test_handler_with_options(self, print_count, drop_all, bulk_create_by, reimbursements, print_):
-        self.command.handle(dataset='reimbursements.xz', batch_size=1, drop=True)
+    @patch('jarbas.core.management.commands.reimbursements.Command.mark_not_updated_reimbursements')
+    def test_handler_with_options(self, mark, print_count, drop_all, create, reimbursements, print_):
+        self.command.handle(dataset='reimbursements.xz', drop=True)
         print_.assert_called_once_with('Starting with 0 reimbursements')
         drop_all.assert_called_once_with(Reimbursement)
-        bulk_create_by.assert_called_once_with(reimbursements, 1)
-
-    @patch('jarbas.core.management.commands.reimbursements.LoadCommand.add_arguments')
-    def test_add_arguments(self, super_add_arguments):
-        parser = MagicMock()
```

Comment:
`MagicMock` is not being used anymore, could you please remove it from the imports?